### ISSUE TITLE
cwltools

### ISSUE THREAD

**Issue by felipeZ** [MD-Studio/cerise-mdstudio-das5#9]
as mention in the [cerise specialization documentation](http://cerise.readthedocs.io/en/develop/specialising.html), `cerise-client` uses [cwltiny](https://github.com/MD-Studio/cerise/blob/8006176fcf72ade21526090fcf2e7e6245e8845d/api/files/cerise/cwltiny.py) as default cwl interpreter but it would be great if we can use some features of CWL like [secondaryFiles](http://www.commonwl.org/v1.0/CommandLineTool.html#CommandInputParameter) that are currently not supported by [cwltiny](https://github.com/MD-Studio/cerise/blob/8006176fcf72ade21526090fcf2e7e6245e8845d/api/files/cerise/cwltiny.py).

**Comment by LourensVeen** [comment]
Good idea, and go ahead, but keep in mind that secondaryFiles is also a Cerise issue, it needs to know (how) to stage them. So just swapping in cwltool won't fully resolve that issue. I've made https://github.com/MD-Studio/cerise/issues/63 for the Cerise side of this.

**Comment by felipeZ** [comment]
Thanks for the clarification! :+1: 
Meanwhile I will play with `cwltools`.
